lib/iex/lib/iex/introspection.ex

```diff
@@ -141,6 +146,18 @@ defmodule IEx.Introspection do
     end
   end
 
+  defp callback_module(mod, fun, arity) do
+    mod.module_info(:attributes)
+    |> Keyword.get_values(:behaviour)
+    |> Stream.concat()
+    |> Stream.filter(fn(module)->
+      callbacks = module.module_info(:attributes)
+      |> Enum.filter_map(&match?({:callback, _}, &1), fn {_, [{t,_}|_]} -> t end)
+      for({f, a} <- callbacks, f == fun, a == arity, do: :ok) != []
```

Comment:
@mgwidmann only one thing though, we should always use `and` for strictly boolean checks.